### DISCUSSION TITLE
Namespace icon for C++

### DISCUSSION THREAD

**Discussion by kthenrique** [SmiteshP/nvim-gps#73] [Ideas]
https://github.com/SmiteshP/nvim-gps/blob/6afd8f7a69e7008dabe048d4fb1ee8e467a72fdf/queries/cpp/nvimGPS.scm#L14

Is there anyway to change namespace icons for C++ without having to change the respective query file?
locally, I'm renaming it to "namespace-name" to be able to have a different icon for namespace.

**Comment by SmiteshP** [comment]
Nope, there isn't any way for changing that without changing the query itself.

**Reply by kthenrique** [reply]
Any objections against changing this at master?

**Reply by SmiteshP** [reply]
I feel classes and namespaces are "similar" enough to share a same icon. I would not like to add another icon for namespaces specifically.
